### ISSUE TITLE
[Bug] 0.38.0.2 version isn't running in my browser.

### ISSUE THREAD

**Issue by Victorio115** [raingart/Nova-YouTube-extension#58]
*Describe the bug*

 - Browser: Brave Browser, [Version 1.48.171 Chromium: 110.0.5481.177 (Official Build) (x86_64)](https://brave.com/latest/)
 - Tampermonkey version 4.18.1
 - Nova version: 0.38.0.2, recently updated.    
  
In Tampermonky appears the script is running, but actually isn't. 
  
![image](https://user-images.githubusercontent.com/50971728/221103416-6b4bf4b5-b952-40d6-a57b-adad789d814d.png)  
  
* I've tried Factory Reseting the extension: No result.  
* I've tried reinstalling the extension: No result.  
* I've tried to load the youtube page in a Private Window: No result.
* The previous version worked with no problems in my browser.  
* I've downgraded the extension to 0.38.0 by [GreasyFork](https://greasyfork.org/en/scripts/433360-nova-youtube): It worked again.  
   
![image](https://user-images.githubusercontent.com/50971728/221108333-46983594-b422-4b9b-997b-4296bbef4bbd.png)

**Comment by raingart** [comment]
try to manually copy the code from openuserjs and replace.

It seems to me that openuserjs gives only part of the script. And that's why it doesn't work.

Compare the ends of the file to see if they are different

![Screenshot_1](https://user-images.githubusercontent.com/13064767/221134451-49137c94-2b4a-4250-8909-fb0676f2bd02.png)

**Comment by raingart** [comment]
sometimes openuserjs gives the full file if you try to install the minified version (while not compresed it)

![image](https://user-images.githubusercontent.com/13064767/221135300-ceadeb44-53ef-480f-a2c1-dff402b58b3c.png)

**Comment by Victorio115** [comment]
Booth versions ended like these...  
  
![image](https://user-images.githubusercontent.com/50971728/221140656-b6ead23f-ecbc-4bfe-bc89-e3e8c9e964ac.png)  

![image](https://user-images.githubusercontent.com/50971728/221140753-c8834860-fd98-4b65-9187-31a77248aef7.png)  
  
While previous version's end looked like this.  
  
![image](https://user-images.githubusercontent.com/50971728/221141032-8c1c411b-4375-47eb-a8e4-dcacdeef29fb.png)

**Comment by raingart** [comment]
openuserjs crashes.
Probably it will be necessary to publish a link to github in openuserjs as an update. Because I don't see any other way to get around the problem.

openuserjs will complain but it's unsafe.
![image](https://user-images.githubusercontent.com/13064767/221142968-b416d74e-591d-4f67-abf0-990889e8153f.png)

greasyfork generally cuts out links to external update sources automatically

Here is an intersting project. How it bypassed greasyforks file size limits
https://greasyfork.org/en/scripts/370418-ppixiv-for-pixiv/code

perhaps if the size of the script is reduced, then there will be no error. If you are well versed in bash and know how to remove part of a line from files.

**Comment by raingart** [comment]
try copying the code and manually replacing it. I think it will help

![image](https://user-images.githubusercontent.com/13064767/221144456-957985ed-3d88-4376-9eb3-6f43f7d606c0.png)

**Comment by Victorio115** [comment]
I copied the source code from OpenUserJS.  
  
![image](https://user-images.githubusercontent.com/50971728/221150057-1c46abb7-fc88-4c96-a627-935d577efd36.png)  
![image](https://user-images.githubusercontent.com/50971728/221149683-7010e18f-a60c-4d87-acba-421e5073ad3b.png)
  
And now the new version is working.  
  
![image](https://user-images.githubusercontent.com/50971728/221149948-87529fd4-42b2-48e7-be90-e252c73ae5b8.png)  
  
Just in case, I kept the version of GreasyFork.  
  
![image](https://user-images.githubusercontent.com/50971728/221150344-0f5b3734-11b5-4ed3-a239-d3ff1dd32c94.png)
  
Thank you.
